frontend/src/services/eventHandlers/hubHandler.js

```diff
@@ -1,12 +1,63 @@
+import { EventBus } from "@/eventBus";
+import { Event } from "@/events";
 import Player from "@/modules/common/entities/player";
 import { store } from "@/store";
 
 // Event handler for Hub API
 export default class HubHandler {
     handle(payload) {
         if (payload.connectedClients != null) {
-            const players = payload.connectedClients.map(client => new Player(client.clientUUID.UUID, client.nickname, client.joinOrder));
+            const players = payload.connectedClients.map(
+                (client) =>
+                    new Player(
+                        client.clientUUID.UUID,
+                        client.nickname,
+                        client.joinOrder
+                    )
+            );
+
+            this.handlePlayerJoined(players);
+            this.handlePlayerLeft(players);
```

Comment:
At this point, we dont' really know if a player joined or left. Probably safe to say the number of players changed, There's a lot of duplicate code between the two funcs, I think they should be combined into one and be called `handlePlayersChanged`